src/app/settings/admin/presets/template.html

```diff
@@ -35,7 +35,7 @@
                     mat-icon-button
                     class="km-clear-input-btn"
                     aria-label="Clear">
-              <i class="km-icon-close"></i>
```

Comment:
This might be a good chance to improve overall styling and use `matSuffix` instead of absolute positioning.

https://material.angular.io/components/form-field/overview#form-field-appearance-variants

Reply:
Probably spend a bit too much time to figure out some css rules to let everything nearly look the same as before...
This causes a new class `km-dropdown-with-suffix` if a dropdown should have a suffix and additionally if it is a filter, which has reduced with `km-filter-with-suffix` class.

Finally noticed following part of the documentation 😅 
`We do not recommend using text prefix and suffixes in the fill and outline variants because the label and input do not have the same alignment. It is therefore impossible to align the prefix or suffix in a way that looks good when compared with both the label and input text.`

However, I commited my changes, would be great if you could have a look. (I'm also fine with reverting it if needed ;) )